### ISSUE TITLE
Next loud failed to upload

### ISSUE THREAD

**Issue by geekie87** [nextcloud/android#7842]
### Steps to reproduce
1. Open next loud app
2. Press + to upload a file
3. Then error popup

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:10

Device model: Oppo f11 pro

Stock or customized system: stock

Nextcloud app version:
Latest
Nextcloud server version:
20.0.4
Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
************ CAUSE OF ERROR ************

java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.
	at android.view.ViewGroup.addViewInner(ViewGroup.java:5274)
	at android.view.ViewGroup.addView(ViewGroup.java:5089)
	at android.view.ViewGroup.addView(ViewGroup.java:5029)
	at android.view.ViewGroup.addView(ViewGroup.java:5002)
	at com.owncloud.android.ui.fragment.OCFileListBottomSheetDialog.onCreate(OCFileListBottomSheetDialog.java:131)
	at android.app.Dialog.dispatchOnCreate(Dialog.java:433)
	at android.app.Dialog.show(Dialog.java:315)
	at com.owncloud.android.ui.fragment.OCFileListFragment.lambda$registerFabListener$2$OCFileListFragment(OCFileListFragment.java:442)
	at com.owncloud.android.ui.fragment.-$$Lambda$OCFileListFragment$aYkoWPoxzAOahKL5AVjT06NhcnM.onClick(Unknown Source:4)
	at android.view.View.performClick(View.java:7257)
	at android.view.View.performClickInternal(View.java:7213)
	at android.view.View.access$3800(View.java:828)
	at android.view.View$PerformClick.run(View.java:27921)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:7830)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1040)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140290
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: OPPO
Device: OP4863
Model: CPH1969
Id: QP1A.190711.020
Product: CPH1969

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 1594883800

**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
This has been fixed with yesterdays update 3.14.3 already avaliable via Google Play, f-droid to folllow during the next days

**Comment by geekie87** [comment]
Thanks 😊👍
